src/Components/ServiceScene/Breakdowns/AddToFiltersButton.tsx

```diff
@@ -16,6 +16,11 @@ export interface AddToFiltersButtonState extends SceneObjectState {
 
 export type FilterType = 'include' | 'reset' | 'exclude' | 'toggle';
 
+export function addAdHocFilter(filter: AdHocVariableFilter, variableName: string, scene: SceneObject) {
+  const type: FilterType = filter.operator === '=' ? 'toggle' : 'exclude';
```

Comment:
Why  in `const type: FilterType = filter.operator === '=' ? 'toggle' : 'exclude';` we are doing `toggle` instead of `include`, as the UI doesn't show that you could toggle it (no highlight). Seems like change of functionality, which I am not sure is intentional.

Reply:
Toggle is a "smarter" behavior when the filter already exists, so the change was intentional. But, since this function is only used by the Table and the table has no "active" status of filters, I better revert this change.

We intentionally use `toggle` for log details, which shows the active state.